hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/validation/ExpiryValidatorImpl.java

```diff
@@ -158,26 +156,21 @@ private boolean hasCompleteAutoRenewSpec(
     /**
      * Helper to validate that the given account number is a valid auto-renew account.
      *
-     * @param shard the account shard to validate
-     * @param realm the account realm to validate
-     * @param num the account number to validate
+     * @param accountID the account id to validate
      * @throws HandleException if the account number is invalid
      */
-    private void validateAutoRenewAccount(final long shard, final long realm, final long num) {
+    private void validateAutoRenewAccount(final AccountID accountID) {
         final var hederaConfig = context.configuration().getConfigData(HederaConfig.class);
-        validateTrue(shard == hederaConfig.shard() && realm == hederaConfig.realm(), INVALID_AUTORENEW_ACCOUNT);
-        if (num == 0L) {
+        validateTrue(
+                accountID.shardNum() == hederaConfig.shard() && accountID.realmNum() == hederaConfig.realm(),
+                INVALID_AUTORENEW_ACCOUNT);
```

Comment:
I am not sure if this is needed 🤔 @tinker-michaelj  thoughts ?

Reply:
Left as it is.